VMEncryption/main/handle.py

```diff
@@ -971,6 +971,24 @@ def handle_encryption(public_settings, encryption_status, disk_util, bek_util, e
                 logger.log('Calling enable for volume type {0}.'.format(volume_type))
                 enable_encryption()
 
+def _is_confidential_temp_disk_encryption(self,logger):
```

Comment:
handle.py does not have class, self is not used here. 
logger is global variable in handle.py class.

Reply:
done. and tested.